### ISSUE TITLE
Question about register 56 in dmpInitialize()

### ISSUE THREAD

**Issue by bpotatog** [jrowberg/i2cdevlib#212]
hi, I'm trying to figure out how libraries of MPU6050 work.
In dmpInitialize(), around line 400
<code>DEBUG_PRINTLN(F("Setting DMP and FIFO_OFLOW interrupts enabled..."));
</code>
<code>setIntEnabled(0x12);
</code>
I checked register 56(MPU6050_RA_INT_ENABLE)
 in MPU6050 register map, bit 4 is configure for FIFO overflow enable.
However, bit 1is reserved.
So I don't understand why bit 4 and bit 1 should both be set.
I changed 0x12 to 0x10 and the dmp just couldn't work.
Is bit 1 used by dmp interrupt?

**Comment by eadf** [comment]
Invensense didn't document every bit. Some info has, however, been reverse engineered.
These are the bits of register MPU6050_RA_INT_ENABLE:
```
#define MPU6050_INTERRUPT_FF_BIT            7
#define MPU6050_INTERRUPT_MOT_BIT           6
#define MPU6050_INTERRUPT_ZMOT_BIT          5
#define MPU6050_INTERRUPT_FIFO_OFLOW_BIT    4
#define MPU6050_INTERRUPT_I2C_MST_INT_BIT   3
#define MPU6050_INTERRUPT_PLL_RDY_INT_BIT   2
#define MPU6050_INTERRUPT_DMP_INT_BIT       1
#define MPU6050_INTERRUPT_DATA_RDY_BIT      0
```
setIntEnabled(0x12) is the same thing as:

`setIntEnabled(1<<MPU6050_INTERRUPT_FIFO_OFLOW_BIT|1<<MPU6050_INTERRUPT_DMP_INT_BIT);`

tl;dr: yes
